### PR TITLE
[CNSMR-3254] Playbook fresh new look

### PR DESCRIPTION
**Pull request by AnnKatF**
**Why?**
Playbook was quite out-dated, it's structure didn't scale very well and was missing a lot of information.

**How?**
Giving a new fresh face to our playbook.
Adding a new cookbook that looks more like a table of contents where everything is organised by topic and it's easy to locale the documents.

### CODE REVIEW

File: Cookbook/README.md [hunk 1]
```diff
@@ -1,62 +1,131 @@
...
-* [Configure PullAssigner on our repos](Technical-Documents/PullAssigners.md)
-* [App Center](Technical-Documents/AppCenter.md)
-* [RubyMine](Technical-Documents/RubyMine.md)
+# Cookbook 🥘
+
+## Onboarding 🚢
+  - Overview
+  - [New hires checklist](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/NewHiresCheckList.md)
+  - Team Roles
+  - Team Structure
```
Comment (sdaniel55) [inline Cookbook/README.md]: Is this the squads table in the root README?

File: Cookbook/README.md [hunk 2]
```diff
@@ -1,62 +1,131 @@
...
-* [Code Signing with Fastlane Match](./Technical-Documents/FastlaneMatch.md)
-* [How to invoke CI jobs from Slack](./Technical-Documents/SlackCIIntegration.md)
-* [Xcode Tips & Tricks](Technical-Documents/XcodeTips.md)
-* [Configure PullAssigner on our repos](Technical-Documents/PullAssigners.md)
-* [App Center](Technical-Documents/AppCenter.md)
-* [RubyMine](Technical-Documents/RubyMine.md)
+# Cookbook 🥘
+
+## Onboarding 🚢
+  - Overview
```
Comment (sdaniel55) [inline Cookbook/README.md]: Do we need these `Overview` points if they don't link to a page?

File: Cookbook/README.md [hunk 3]
```diff
@@ -1,62 +1,131 @@
...
+  - [New hires checklist](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/NewHiresCheckList.md)
```
Comment (sdaniel55) [inline Cookbook/README.md]: [Question] Any reason why we don't use relative paths anymore?

File: Cookbook/README.md [hunk 4]
```diff
@@ -1,62 +1,131 @@
...
+  - [Overview](https://github.com/babylonhealth/ios-playbook/blob/master/Interview/README.md) 🛠
+  - [Software Engineer Technical test](https://github.com/babylonhealth/ios-playbook/blob/master/Interview/demo.md)
+  - [UI Automation Engineer Technical Test](https://github.com/babylonhealth/ios-playbook/blob/master/Interview/AutomationExercise.md)
+  - Pair programming session
+  - [Technical questions](https://github.com/babylonhealth/ios-playbook/blob/master/Interview/questions.md)
+  - Cultural fit questions
+  - Interview presentation
+
+## Tooling 🛠
+  - [XCode Tips & Tricks](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/XcodeTips.md)
```
Comment (sdaniel55) [inline Cookbook/README.md]:
```suggestion
  - [Xcode Tips & Tricks](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/XcodeTips.md)
```

**Review by danspndl — APPROVED** [review]
Looks good, left a few questions

File: Cookbook/README.md (hunk 1 above)
Comment (AnnKatF) [reply Cookbook/README.md]: This should be to explain our structure in terms of Chapter, Tribes and Squads.

Comment (sdaniel55) [reply Cookbook/README.md]: I think there's already a page for this, it might not be public though :/

Comment (AnnKatF) [reply Cookbook/README.md]: If it's the one I know about is the company internal one that has all the tribes and squads. This is more to explain the concept of Chapter, Tribe and Squad.

File: Cookbook/README.md (hunk 3 above)
Comment (AnnKatF) [reply Cookbook/README.md]: No reason really. I just copy pasted this from the private one 😅Will fix it! 👍

File: Cookbook/README.md (hunk 2 above)
Comment (AnnKatF) [reply Cookbook/README.md]: Yeah...we don't need them. Will remove it!

File: Cookbook/README.md [hunk 5]
```diff
@@ -1,64 +1,129 @@
...
+  - [Computer set-up](./Technical-Documents/NewHiresCheckList.md#install-prerequisites)
+  - [Initial tasks](./Technical-Documents/NewHiresCheckList.md#whats-next)
+
+## Process 🚓
+  - [Day-to-day in the iOS Team FAQ](./Technical-Documents/WorkingAtBabylon.md)
+  - [Code owners](./Proposals/CODEOWNERS.md)
+  - [Writing proposals](./Technical-Documents/WritingAProposal.md)
+  - Writing Documentation
+  - Development process
+  - [Jira](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/JIRA.md)
```
Comment (MichalTKwiecien) [inline Cookbook/README.md]: It's not merged yet but will be there soon (PR is already open)
```suggestion
  - [Jira](./Technical-Documents/JIRA.md)
```

**Review by MichalTKwiecien — APPROVED** [review]

**Pull request merged**